qa/cephfs/conf/client.yaml

```diff
@@ -5,5 +5,6 @@ overrides:
         client mount timeout: 600
         debug ms: 1
         debug client: 20
+        debug objecter: 20
```

Comment:
Is this required for this change?

Reply:
Not necessary for this but handy to debug for the test that would get added with https://tracker.ceph.com/issues/68870

Reply:
yeh, let's move it into the changes for that tracker then? I'm fine with this here too, but I think it is a bit more relevant there.

Reply:
Sure I will rebase taking this out.

Reply:
Done.